### ISSUE TITLE
Having unique dialog for characters dependent on if there is a crisis or not

### ISSUE THREAD

**Issue by Micky1503** [Spexify/emergenCity#140]
"Es ist ein bisschen befremdlich, wenn am ersten Tag noch gar kein Prompt zu einer Krise kam, aber schon mehrere Menschen in der Stadt besorgt darüber sprechen und Zutaten mit mir tauschen (cool) wollen."

**Comment by Micky1503** [comment]
I don't think we should have dialog dependent on the particular scenario, but dialog dependent on IF there is a crisis or not. When there is no crisis (you have water, electricity, etc.) and Agathe says "Man sieht ja gar keine Leute in einer Krise wie dieser", then, well. Reads a bit awkward

**Comment by Spexify** [comment]
Capabilities for this now existe with the new System https://github.com/Spexify/emergenCity/commit/17dd9769b386472c01f83df6eb78d4b9bb1c02a9, but the dialogues are not yet migrated.
